### ISSUE TITLE
User Guides

### ISSUE THREAD

**Issue by gitr0n1n** [ethereumclassic/ethereumclassic.github.io#390]
Need Guides and Content.

https://deploy-preview-351--ethereumclassic.netlify.com/knowledge/guides

**Comment by gitr0n1n** [comment]
Fix Link:

"What is and how Does Ethereum Classic Work in Layman’s Terms?"
link to: https://deploy-preview-351--ethereumclassic.netlify.com/knowledge/what-is-etc/

**Comment by gitr0n1n** [comment]
Change and Fix Link
"Node guides."
to
"Ethereum Classic Client guides"

Add Client Guide:
"Client: Core-Geth
Description: Core-Geth is a distribution of go-etheruem with many flavors.
Link: https://core-geth.org/install "

**Comment by gitr0n1n** [comment]
"Mining guides."
Fix Link

Add Miner Guide:
"Miner: BMiner
Description: Bminer is a highly optimized cryptocurrency miner that runs on modern AMD / NVIDIA GPUs. Bminer is one of the fastest publicly available miners today -- we use various techniques including tiling and pipelining to realize the full potentials of the hardware.
Link: https://www.bminer.me/ "

Add Miner Guide:
"Client: EthMiner
Description: Ethminer is an Ethash GPU mining worker: with ethminer you can mine every coin which relies on an Ethash Proof of Work thus including Ethereum, Ethereum Classic, Metaverse, Musicoin, Ellaism, Pirl, Expanse and others. This is the actively maintained version of ethminer. It originates from cpp-ethereum project (where GPU mining has been discontinued) and builds on the improvements made in Genoil's fork. See FAQ for more details.
Link: https://github.com/ethereum-mining/ethminer "

Add Miner Guide:
"Client: MinerGate xFast GUI Miner
Description: MinerGate xFast GUI Miner is an easy-to-use yet high-performance mining software. Based on the new code architecture, the app shows impressive hashing output, making your mining experience even more efficient.
Link: https://minergate.com/downloads "

Add Client Guide:
"Client: Core-Geth
Description: Core-Geth is a distribution of go-etheruem with many flavors.
Link: https://core-geth.org/install "

**Comment by gitr0n1n** [comment]
Block Explorer guides.

Add:
Explorer: TokenView
Link: https://tokenview.com/en/learn

**Comment by gitr0n1n** [comment]
Remove raw links:

"https://wiki.hyperledger.org/display/BESU/Building+from+source
https://besu.hyperledger.org/en/stable/
https://pegasys.tech/solutions/hyperledger-besu/"

**Comment by hitchcott** [comment]
Right now I'm thinking that we might as well remove the user guides section, as all of these links are covered on other parts of the website.

In lieu of this page, we could add some links to the bottom of the _What is Ethereum Classic_ page?
